components/ShoppingItem.js

```diff
@@ -1,24 +1,57 @@
 import styled from "styled-components";
 import Link from "next/link";
+import { useState } from "react";
+import Delete from "@/public/icons/delete.svg";
+
+export default function ShoppingItem({
+  id,
+  quantity,
+  name,
+  category,
+  onDeleteItem,
+}) {
+  const [isDeleteConfirmVisible, setIsDeleteConfirmVisible] = useState(false); // Zustand für den Bestätigungsdialog
```

Comment:
You could delete the comments. We were told by the coaches that we should do that...